.github/workflows/gh-actions-release.yml

```diff
@@ -35,13 +35,8 @@ jobs:
             OpenGL_GL_PREFERENCE: 'GLVND'
             ENABLE_PIE: 'OFF'
             ARTIFACT_EXT: 'deb'
-          - FROM:     'ubuntu:focal'
-            COMPILER: 'clang'
-            OpenGL_GL_PREFERENCE: 'GLVND'
-            ENABLE_PIE: 'OFF'
-            ARTIFACT_EXT: 'deb'
           # Source tarballs only
-          - FROM:     'ubuntu:focal'
+          - FROM:     'ubuntu:jammy'
```

Comment:
Jammy still carries boost 1.74. We need 1.75 for boost json.

Reply:
Oh, gotcha. All right. I can take that out.

Reply:
Fixed.